pymc_marketing/clv/models/basic.py

```diff
@@ -47,6 +47,19 @@ def __init__(
         non_distributions: list[str] | None = None,
     ):
         model_config = model_config or {}
+
+        deprecated_keys = [key for key in model_config if key.endswith("_prior")]
```

Comment:
Let's add a TODO to remove these codelines in a future release. I'd say 2025-Q4 or early 2026 will give users enough time to update their codebases.